README.md

```diff
@@ -76,23 +50,26 @@ workbook/0n-section-name.asciidoc // Contains the part of the workbook matching
 ```
 
 After material is finished the scripts will be used to film videos.
-Videos, articles, and workbooks will be reviewed by O'Reilly's professional editors and then published at _learning.oreilly.com_ and also at _innersourcecommons.org_.
+Each video should be about 5min in length.
+Each article should be about a page long.
```

Comment:
A4, Letter, piece of parchment, 4k screen on 10 point, cowhide?
I often use a metric of "xyz words/letters" to describe a length of an article independent of its rendering. 
We could use 1000-2000 letters as a starter.
```suggestion
Each article should be about 1000-2000 letters, i.e. about a page, long.
```

Reply:
Even though people _could_ be confused, do you think that people _will_ be confused?

Reply:
> Even though people _could_ be confused, do you think that people _will_ be confused?

Let me mention this lovely German invention I found out about today, the [Normseite](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fde.m.wikipedia.org%2Fwiki%2FNormseite). Obviously everything needs to be standardized into oblivion or else! :wink: 

On a more serious note: I'm fine either way. If someone has questions they can still ask.

Reply:
OK I'm going to leave out for now.